Source/WebCore/rendering/RenderLayerModelObject.h

```diff
@@ -101,6 +101,7 @@ class RenderLayerModelObject : public RenderElement {
     RenderSVGResourceClipper* svgClipperResourceFromStyle() const;
     RenderSVGResourceMasker* svgMaskerResourceFromStyle() const;
 
+    void paintSVGClippingMask(PaintInfo&, const FloatRect& objectBoundingBox) const;
     void paintSVGClippingMask(PaintInfo&) const;
```

Comment:
Do we need this accesor? Could get rid of it and only keep one.

Reply:
Makes sense, we now need the FloatRect parameter.